src/Dofs/DofHandler.jl

```diff
@@ -582,18 +582,15 @@ element B in the example.
 
 In addition, we also have to preverse the ordering at each dof location.
 
-For more details we refer to [1] as we follow the methodology described therein.
-
-[1] Scroggs, M. W., Dokken, J. S., Richardson, C. N., & Wells, G. N. (2022).
-    Construction of arbitrary order finite element degree-of-freedom maps on
-    polygonal and polyhedral cell meshes. ACM Transactions on Mathematical
-    Software (TOMS), 48(2), 1-23.
+For more details we refer to Scroggs et al. [Scroggs2022](@cite) as we follow the methodology
+described therein.
 
-    !!!TODO Citation via DocumenterCitations.jl.
-
-    !!!TODO Investigate if we can somehow pass the interpolation into this function in a typestable way.
+# References
+ - [Scroggs2022](@cite) Scroggs et al. ACM Trans. Math. Softw. 48 (2022).
```

Comment:
For future reference: Is there something stopping us from generating this one automatically, similar to the reference generator in the examples?

Reply:
Which reference generator?

Reply:
https://github.com/Ferrite-FEM/Ferrite.jl/blob/master/docs/src/literate-gallery/topology_optimization.jl#L530-L534 where, for example, we put something else than `Pages`. Or even automate adding the references in docstrings with another plugin.

Reply:
Ah, okay. No, because this is in a docstring and there can't really be anything dynamic there since you want it to look nice also in the REPL help mode and such. See e.g. https://github.com/JuliaDocs/DocumenterCitations.jl/issues/21 for some related discussion.